pkg/cgroup/cgroup.go

```diff
@@ -0,0 +1,292 @@
+package cgroup
+
+import (
+	"bufio"
+	"fmt"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"syscall"
+
+	"github.com/aquasecurity/tracee/pkg/mount"
+)
+
+type Cgroup interface {
+	initialize() error
+	Destroy() error
+	GetMountPoint() string
+	GetDefaultHierarchy() int
+	IsV1() bool
```

Comment:
If we go for the approach I suggested, we may need to also add `IsV2() bool` for the combined implementation (if it's a necessary one).